microsoft-edge/devtools-guide-chromium/javascript/overrides.md

```diff
@@ -76,19 +76,19 @@ The file is automatically saved on your computer.  If you refresh the file, the
 
 Changing the webpage styles persistently by editing a file in your overrides folder:
 
-:::image type="content" source="../media/javascript-overrides-changing-styles.msft.png" alt-text="Changing the webpage styles persistently by editing a file in your overrides folder." lightbox="../media/javascript-overrides-changing-styles.msft.png":::
+![Changing the webpage styles persistently by editing a file in your overrides folder.](../media/javascript-overrides-changing-styles.msft.png)
```

Comment:
@captainbrosset  - sometimes GitHub highlights for no discernible reason.  pls inspect tagging characters for validity.

Reply:
I'll push a new commit and see if this still happens.

Reply:
Still happens in this file only. I have no idea why. I'm almost tempted to say this is a github bug. The red highlight normally appears for syntax errors, but there's obviously no error here.
I've also tried to retype the entire thing by hand, and git saw no changes. So I think we should ignore this.